tests/plugins/kind-projector/KindProjector.scala

```diff
@@ -1,13 +1,12 @@
 package anx
```

Comment:
This file doesn't seem to be using kind-projector anymore, which seems to defeat its purpose

Reply:
Good catch. I was playing around with fixing this test and left it in a passing, but useless state. This is fixed.

Reply:
discussed offline; @coreywoodfield said it looks good